lib/skeleton/config/scripts/ios/ci_ios_main.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# Input variables:
+# $1 - The parameters of what test to execute. Example "- @dev" or "" for nothing
+
+ambiente="CI"
+
+cd $WORKSPACE
+
+if [ $ambiente == "CI" ]
+then
+	export IOS_WORKSPACE=$WORKSPACE/../isp-participantemobile-ios-master
+	ruby ./config/scripts/ios/build_app.rb jenkins simulator
+	ruby ./config/scripts/ios/build_app.rb jenkins devices
+fi
+export LANG=en_US.UTF-8
+export LANGUAGE=en_US:en
+
+
+#sh $WORKSPACE/config/scripts/ios/run_tests_all_devices.sh ./app/build/ios/simulator/Gebsa-cal.app ./app/build/ios/simulator/Gebsa-cal.app
+sh $WORKSPACE/config/scripts/ios/run_tests_all_devices.sh null null "$1"
```

Comment:
Are you sure the first two parameters must be null?
You could export the APP_BUNDLE_PATH variable created in the build_app.rb and save it to use it here.